docs/DTW_MP_trial.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.        r,
</pre><p>What is <code>r</code>?? Is <code>r &lt;= m</code>  always <code>True</code> or can <code>r</code> be greater than <code>m</code>?     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
We need some preprocess for limitation of `r`.